### PR TITLE
added README

### LINKED ISSUE #13: Add Readme
Add Readme file explaining how to use and setup the project 

Add pipeline badges

### PR DESCRIPTION
**Pull request by Willian1661**
add a simple README

close #13

### CODE REVIEW

**Comment by Willian1661** [comment]
@eerison 
in this one commit i used:
- git stash save "message" (to save my working directory)
-> modified the readme
- git add .
- git commit
- git push
- git stash pop (to recover the saved directory witl the staged files)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,43 @@
...
+At this point one more thing you should run in your terminal, are the tests. Follow the next steps to run Jest.
+
+## Runnig test with Jest
+You can run the tests with `npm run test` command line, it will run `jest --coverage` and generate a `/coverage` folder and it'll show this in your terminal:
+
+<img src="src/assets/images/show_test_pass.png" width="240">
+
+Now, since we have our page builded and tests passed we just need to run Eslint in our code to maintain a pattern.
+Run `npm run lint:fix` command line, it'll trigger eslint only on `src/.` , but this can be changed on `.eslintrc.js` configuration.
+it will return all the code that have to be fixed if needed and fix the errors.
```
Comment (eerison) [inline README.md]: Add a calculator screenshot

Comment (Willian1661) [reply README.md]: Ok, but you merged the branch with "final appearance", so i have to update my master first and then screenshot?to get the changes i did with that branch.

**Comment by eerison** [comment]
Those images doesn't make part of your code, put this inside of a `doc` folder in the root path

File: README.md [hunk 2]
```diff
@@ -31,15 +31,15 @@ Then, if you want to automatically save your changes you need to run and after t
 
 ### You just successfully generated your page!
 
-<img src="src/assets/images/calculator.png" width="240">
```
Comment (eerison) [inline README.md]: if you want to use pictures, it's ok.

But put it inside of docs, because those images are related with doc

Comment (Willian1661) [reply README.md]: no problem, i forgot that you've said the folder name doc, it's like what? documents?

Comment (eerison) [reply README.md]: doc of documentation

Comment (Willian1661) [reply README.md]: ok done

**Pull request merged**